code/portal_info.js

```diff
@@ -266,7 +266,7 @@ window.getPortalMitigationDetails = function(d,linkCount) {
   // mitigation is limited to 95% (as confirmed by Brandon Badger on G+)
   mitigation.total = Math.min(95, mitigation.shields+mitigation.links);
 
-  mitigation.excess = (mitigation.shields+mitigation.links) - mitigation.total;
+  mitigation.excess = Math.round(10 * ((mitigation.shields+mitigation.links) - mitigation.total)) / 10;
```

Comment:
```suggestion
  var excess = (mitigation.shields+mitigation.links) - mitigation.total;
  mitigation.excess = Math.round(10*excess) / 10;
```

And better write add more info to commit message, like this:

> JS float precision is not good, so without rounding we'd get `125.8 - 95 = 30.799999999999997`
  (More background info: https://floating-point-gui.de/)